src/Account.ts

```diff
@@ -1,8 +1,36 @@
-const MultiSig = require("./Signers/MultiSig.js");
-const BNSigner = require("./Signers/BNSigner.js");
-const SecpSigner = require("./Signers/SecpSigner.js");
-// Below import for intellisense and type support on jsdoc
-const Wallet = require('./Wallet.js'); //eslint-disable-line
+import MultiSig from "./Signers/MultiSig";
+import BNSigner from "./Signers/BNSigner";
+import SecpSigner from "./Signers/SecpSigner";
+import { WalletParams } from './Wallet';
+
+
+// TODO Move this to Multisig.js/BNSigner.js
+export interface Signer {
+    multiSig: () => {};
+    getAddress: () => {};
+    addPublicKeys: () => {};
+}
+
+export interface Utxo {
+    DataStores: Array<any>;
+    ValueStores: Array<any>;
+    ValueStoreIDs: Array<any>;
+    DataStoreIDs: Array<any>;
+    Value: string | number | bigint;
+}
+
+export interface AccountObject {
```

Comment:
I tried `Account` as the interface name but it conflicts with the class name.
Do you think I'm missing something here?

Reply:
This naming scheme is rooted in use calling the class 'Account' what it is as opposed to something like 'AccountManager' since it manages accounts, it isn't an account object itself imo. Due to this I'm okay with calling it AccountObject or renaming the class to AccountManager, but that has more overhaul and breaks integrations -- Thoughts?